cli/azd/internal/appdetect/maven_command.go

```diff
@@ -12,8 +12,16 @@ import (
 	"strings"
 )
 
-func getMvnCommand(pomPath string) (string, error) {
-	mvnwCommand, err := getMvnwCommandInProject(pomPath)
+func getMvnCommand() (string, error) {
```

Comment:
Hi, @saragluna . You can use this func to get maven command in the hook related code.